tests/CMakeLists.txt

```diff
@@ -249,6 +248,53 @@ else()
     message(STATUS "Most tests are disabled because AVIF_ENABLE_GTEST is OFF.")
 endif()
 
+################################################################################
+# Experimental FuzzTest support (Linux only)
+
+if(AVIF_ENABLE_GTEST AND AVIF_ENABLE_FUZZTEST)
+    if(AVIF_LOCAL_FUZZTEST)
+        # Run ext/fuzztest.cmd first.
+        # Recommended top-level CMake options:
+        #   -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DAVIF_ENABLE_WERROR=OFF
+        # Reproducing a failure can be done by setting the environment variable
+        #   FUZZTEST_REPLAY=/path/to/repro_file.test
+        # and running one of the targets below.
+        # See https://github.com/google/fuzztest/blob/main/doc/quickstart-cmake.md
+        add_subdirectory(${CMAKE_SOURCE_DIR}/ext/fuzztest ${CMAKE_SOURCE_DIR}/ext/fuzztest/build.libavif EXCLUDE_FROM_ALL)
+        # FuzzTest bundles GoogleTest so no need for the targets below to link to GTEST_BOTH_LIBRARIES.
+    else()
+        message(FATAL_ERROR "fuzztest: Installed FuzzTest is not supported, please set AVIF_LOCAL_FUZZTEST=ON or AVIF_ENABLE_FUZZTEST=OFF")
+    endif()
+
+    add_executable(avif_fuzztest_dec_incr gtest/avif_fuzztest_helpers.cc gtest/avif_fuzztest_dec_incr.cc)
+    target_link_libraries(avif_fuzztest_dec_incr PRIVATE avifincrtest_helpers)
+    link_fuzztest(avif_fuzztest_dec_incr)
+    gtest_discover_tests(avif_fuzztest_dec_incr)
+    add_test(NAME avif_fuzztest_dec_incr COMMAND avif_fuzztest_dec_incr)
+
+    add_executable(avif_fuzztest_enc_dec gtest/avif_fuzztest_helpers.cc gtest/avif_fuzztest_enc_dec.cc)
+    target_link_libraries(avif_fuzztest_enc_dec PRIVATE aviftest_helpers)
+    link_fuzztest(avif_fuzztest_enc_dec)
+    gtest_discover_tests(avif_fuzztest_enc_dec)
+    add_test(NAME avif_fuzztest_enc_dec COMMAND avif_fuzztest_enc_dec)
+
+    if(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
```

Comment:
Should we also check `AVIF_ENABLE_EXPERIMENTAL_AVIR`?

Reply:
When `coni` is tested in `avif_fuzztest_enc_dec_experimental`, sure. It is not the case yet.